x-pack/test/api_integration/apis/ml/results/get_category_examples.ts

```diff
@@ -0,0 +1,198 @@
+        {
+          function: 'count',
+          by_field_name: 'mlcategory',
+          partition_field_name: PARTITION_FIELD_NAME,
+        },
+      ],
+      influencers: ['mlcategory'],
+    },
+    analysis_limits: { model_memory_limit: '26MB' },
+    data_description: { time_field: '@timestamp', time_format: 'epoch_ms' },
+    model_plot_config: { enabled: false, annotations_enabled: true },
+    model_snapshot_retention_days: 10,
+    daily_model_snapshot_retention_after_days: 1,
+    allow_lazy_open: false,
+  };
+  // @ts-expect-error not full interface
+  const testDatafeedConfig: Datafeed = {
+    datafeed_id: `datafeed-${jobIdSpace1}`,
+    indices: ['ft_module_sample_logs'],
+    job_id: jobIdSpace1,
+    query: { bool: { must: [{ match_all: {} }] } },
+  };
+
+  const expectedCategoryExamples1 = {
+    '1': [
+      '130.246.123.197 - - [2018-07-22T03:26:21.326Z] "GET /beats/metricbeat HTTP/1.1" 200 6850 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1"',
+    ],
+  };
+  const expectedCategoryExamples3 = {
+    '1': [
+      '130.246.123.197 - - [2018-07-22T03:26:21.326Z] "GET /beats/metricbeat HTTP/1.1" 200 6850 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1"',
+      '130.246.123.197 - - [2018-07-22T03:26:21.326Z] "GET /beats/metricbeat_1 HTTP/1.1" 200 6850 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1"',
+      '223.87.60.27 - - [2018-07-22T00:39:02.912Z] "GET /elasticsearch/elasticsearch-6.3.2.deb HTTP/1.1" 200 6219 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:6.0a1) Gecko/20110421 Firefox/6.0a1"',
+    ],
+  };
+
+  async function getCategoryExamples(
+    jobId: string,
+    categoryIds: string[],
+    maxExamples: number,
+    user: USER,
+    expectedStatusCode: number,
+    space?: string
+  ) {
+    const { body } = await supertest
+      .post(`${space ? `/s/${space}` : ''}/api/ml/results/category_examples`)
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(COMMON_REQUEST_HEADERS)
+      .send({ jobId, categoryIds, maxExamples })
+      .expect(expectedStatusCode);
+
+    return body;
+  }
+
+  describe('get category_examples', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/module_sample_logs');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await spacesService.create({ id: idSpace1, name: 'space_one', disabledFeatures: [] });
+      await spacesService.create({ id: idSpace2, name: 'space_two', disabledFeatures: [] });
+
+      await ml.api.createAndRunAnomalyDetectionLookbackJob(
+        // @ts-expect-error not full interface
+        testJobConfig,
+        testDatafeedConfig,
+        idSpace1
+      );
+    });
+
+    after(async () => {
+      await spacesService.delete(idSpace1);
+      await spacesService.delete(idSpace2);
+      await ml.testResources.deleteIndexPatternByTitle('ft_module_sample_logs');
```

Comment:
I think we're not creating an index pattern / data view as part of this test, so we don't need to clean it up.

Reply:
Updated in https://github.com/elastic/kibana/pull/124710/commits/7634363c0ce8420f1ce649f17012731ecce30cae